test/integration/services/quotes/sources/quote-sources.spec.ts

```diff
@@ -104,6 +104,18 @@ describe.skip('Quote Sources [External Quotes]', () => {
       });
 
       describe('Sell order', () => {
+        quoteTest({
+          test: Test.SELL_RANDOM_ERC20_TO_STABLE,
+          when: 'swapping 100 random token to stable',
+          request: {
+            sellToken: RANDOM_ERC20,
+            buyToken: nativeToken,
+            order: {
+              type: 'sell',
+              sellAmount: parseUnits('100', 18),
```

Comment:
What would you say if we change this to `unitsToSell: number`, and then, inside the test, we can do `parseUnits('${unitsToSell}', sellToken.decimals)`
So we don't have to worry about decimals here

Reply:
That was my initial thought, but I suck at typing, so I couldn't achieve it 😬

Reply:
I'll do it